ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -190,6 +190,15 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
             _contentView = newValue
             if let contentView = _contentView {
                 containerView.addArrangedSubview(contentView)
+                // Need to set constraints to prevent view getting cut off from UIPopoverArrowDirection in iPad.
+                if !showsResizingHandle {
+                    NSLayoutConstraint.activate([
```

Comment:
you don't put anchor contraints for subviews inside the uistackview. that conflicts with constraints automatically added by the stackview.. it seems like containerview constraints is wrong or your uistackview setup is incorrect. maybe try insetsLayoutMarginsFromSafeArea.